### ISSUE TITLE
UD-Grid Sorting is Case Sensitive

### ISSUE THREAD

**Issue by awickham10** [ironmansoftware/universal-dashboard#1048]
#### Describe the Issue
When sorting mixed case values in a column it appears that it is sorting uppercase alphabetical first and then lowercase alphabetical.

#### To Reproduce
1. Start dashboard with a grid.
2. Click the header to sort.

#### Expected behavior
Case should be ignored when sorting, or at least have the option to ignore case when sorting.

#### Screenshots
![image](https://user-images.githubusercontent.com/7840633/63712889-9830ab80-c80c-11e9-92b1-7dc69478ec48.png)

#### Script Sample
```powershell
Start-UDDashboard -Port 8080 -Force -Content {
    New-UDDashboard -Title 'Test' -Content {
        New-UDGrid -Title 'Data' -Endpoint {
            @(
                @{Header = 'A'},
                @{Header = 'B'},
                @{Header = 'C'},
                @{Header = 'a'},
                @{Header = 'b'},
                @{Header = 'c'}
            )    | Out-UDGridData
        }
    }
}
```

#### Version Information
- **Operating System**: Windows 10
- **PowerShell Version**: 5.1
- **Universal Dashboard Version**: 2.5.3
- **UD Hosting Method**: PowerShell

**Comment by adamdriscoll** [comment]
This issue has been mentioned on **Ironman Software Forums**. There might be relevant details there:

https://forums.universaldashboard.io/t/case-sensitive-udgrid-sorting/2152/1

**Comment by adamdriscoll** [comment]
Duplicate of #471 - We will use that to track. Targeting a fix for 2.9.

**Comment by awickham10** [comment]
Is this actually a duplicate? My issue was for the sort order of data and the other issue is for case sensitivity with property names isn't it?
